jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/CommonUtils.java

```diff
@@ -42,4 +42,16 @@ public static Long getProcessInstanceId(Command<?> command) {
         return null;
 		
 	}
+	
+	// to compensate https://hibernate.atlassian.net/browse/HHH-8091 add empty element to the roles in case it's empty
+	public static List<String> getAuthenticatedUserRoles(IdentityProvider identityProvider) {
+        List<String> roles = identityProvider != null ? identityProvider.getRoles() : new ArrayList<>();
```

Comment:
Seems to be a compilation issue here, new ArrayList<>() should be new ArrayList<String>()